component/src/main/java/io/siddhi/extension/io/grpc/util/ServiceConfigs.java

```diff
@@ -102,34 +111,43 @@ public ServiceConfigs(OptionHolder optionHolder, SiddhiAppContext siddhiAppConte
                 this.sequenceName = urlPathParts.get(GrpcConstants.PATH_SEQUENCE_NAME_POSITION);
             }
         }
-
+        //Validates and enables SSL feature
         if (optionHolder.isOptionExists(GrpcConstants.ENABLE_SSL)) {
             isSslEnabled = Boolean.parseBoolean(optionHolder.validateAndGetOption(GrpcConstants.ENABLE_SSL).getValue());
+        } else {
+            isSslEnabled = true;
         }
-
-        if (isSslEnabled && !optionHolder.isOptionExists(GrpcConstants.TRUSTSTORE_FILE)) {
-            truststoreFilePath = GrpcConstants.DEFAULT_TRUSTSTORE_FILE_PATH;
-            truststorePassword = GrpcConstants.DEFAULT_TRUSTSTORE_PASSWORD;
-            truststoreAlgorithm = GrpcConstants.DEFAULT_TRUSTSTORE_ALGORITHM;
-        }
-
-        if (optionHolder.isOptionExists(GrpcConstants.TRUSTSTORE_FILE)) {
-            truststoreFilePath = optionHolder.validateAndGetOption(GrpcConstants.TRUSTSTORE_FILE).getValue();
-            if (optionHolder.isOptionExists(GrpcConstants.TRUSTSTORE_PASSWORD)) {
-                truststorePassword = optionHolder.validateAndGetOption(GrpcConstants.TRUSTSTORE_PASSWORD)
-                        .getValue();
-            }
-            truststoreAlgorithm = optionHolder.validateAndGetOption(GrpcConstants.TRUSTSTORE_ALGORITHM).getValue();
-            tlsStoreType = optionHolder.getOrCreateOption(GrpcConstants.TLS_STORE_TYPE,
-                    GrpcConstants.DEFAULT_TLS_STORE_TYPE).getValue();
-        }
-
+        //retrieves KeyStore File
         if (optionHolder.isOptionExists(GrpcConstants.KEYSTORE_FILE)) {
             keystoreFilePath = optionHolder.validateAndGetOption(GrpcConstants.KEYSTORE_FILE).getValue();
             keystorePassword = optionHolder.validateAndGetOption(GrpcConstants.KEYSTORE_PASSWORD).getValue();
             keystoreAlgorithm = optionHolder.validateAndGetOption(GrpcConstants.KEYSTORE_ALGORITHM).getValue();
             tlsStoreType = optionHolder.getOrCreateOption(GrpcConstants.TLS_STORE_TYPE,
                     GrpcConstants.DEFAULT_TLS_STORE_TYPE).getValue();
+        } else {
+            keystoreFilePath = configReader.readConfig(GrpcConstants.KEYSTORE_FILE,
+                    GrpcConstants.DEFAULT_KEYSTORE_FILE);
+            keystorePassword = configReader.readConfig(GrpcConstants.KEYSTORE_PASSWORD,
+                    GrpcConstants.DEFAULT_KEYSTORE_PASSWORD);
+            keystoreAlgorithm = configReader.readConfig(GrpcConstants.KEYSTORE_ALGORITHM,
+                        GrpcConstants.DEFAULT_KEYSTORE_ALGORITHM);
+            tlsStoreType = optionHolder.getOrCreateOption(GrpcConstants.TLS_STORE_TYPE,
+                    GrpcConstants.DEFAULT_TLS_STORE_TYPE).getValue();
+        }
+        //retrieves Truststore file
+        if (isSslEnabled && optionHolder.isOptionExists(GrpcConstants.SYS_TRUSTSTORE_FILE_PATH)) {
+            truststoreFilePath = optionHolder.validateAndGetOption(GrpcConstants.SYS_TRUSTSTORE_FILE_PATH).getValue();
```

Comment:
Keys in the annotation cannot be in camel case, change it to truststore.file.path in here only. The ones read from config reader will be in camel case